docs/code-quality/ca1304.md

```diff
@@ -74,6 +74,35 @@ This example produces the following output:
 06/04/1900 12:15:12
 ```
 
+## Configurability
```

Comment:
Please move the section above the examples and also add a small sentence about configurability at the end of `Cause` section, with a hyperlink to here.